pkgs/applications/video/droidcam/default.nix

```diff
@@ -30,7 +39,7 @@ stdenv.mkDerivation rec {
 
   postPatch = ''
     substituteInPlace src/droidcam.c \
-      --replace "/opt/droidcam-icon.png" "$out/share/icons/hicolor/droidcam.png"
+      --replace "/opt/droidcam-icon.png" "$out/share/pixmaps/droidcam.png"
```

Comment:
The correct location is `share/icons/hicolor/96x96/apps`, pixmaps is a [legacy fallback](https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html#directory_layout).